types/marked-terminal/marked-terminal-tests.ts

```diff
@@ -1,4 +1,4 @@
-import * as marked from 'marked';
+import { marked } from 'marked';
```

Comment:
```suggestion
import * as marked from 'marked';
```

Reply:
The default export was removed in [marked@v4.0.0](https://github.com/markedjs/marked/releases/tag/v4.0.0)

Reply:
This will fail with the error:

```
Type 'typeof import("/home/runner/work/DefinitelyTyped/DefinitelyTyped/types/marked-terminal/node_modules/marked/lib/marked")' has no call signatures.
```